packages/twenty-server/src/database/typeorm/typeorm.service.ts

```diff
@@ -89,6 +89,11 @@ export class TypeORMService implements OnModuleInit, OnModuleDestroy {
         ? ['query', 'error']
         : ['error'],
       schema,
+      ssl: this.environmentService.get('PG_SSL_ALLOW_SELF_SIGNED')
```

Comment:
this datasource is not used as we have disable the 1 datasource per workspace pattern which we would need if we want to introduce 1 user per workspace (pg_graphql needs it for performance + great for security/privacy). Likely to be deprecated mid term with twenty orm effort